packages/bpk-docs/src/pages/BannerAlertsPage/bpk-banner-alerts-page.scss

```diff
@@ -19,7 +19,7 @@
 @import '~bpk-mixins/index';
 
 .bpk-banner-alerts-page {
-  &__component:not(:last-child) {
-    margin-bottom: $bpk-spacing-base;
+  &__component {
+    margin-top: $bpk-spacing-base;
```

Comment:
How come this has changed?

Reply:
because I added the reset button, this is now unessessary as we want the top one to be paded too. Will possible revert when I make the reset button appear / disapear etc

Reply:
Ahh ok!